src/service/interfaces.ts

```diff
@@ -34,6 +34,7 @@ export interface IContactInfoForm {
 export type FormPassSingleProfile = {
 	id: number;
 	saved: boolean;
+	clientId: number | undefined;
```

Comment:
`clientId?: number` may it be suitable for you?

Reply:
i fix it